src/main/java/com/chessmates/data/ItemRepositoryImpl.groovy

```diff
@@ -0,0 +1,53 @@
+package com.chessmates.data
+
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient
+import com.amazonaws.services.dynamodbv2.document.DynamoDB
+import com.amazonaws.services.dynamodbv2.document.Item
+import com.amazonaws.services.dynamodbv2.document.Table
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Repository
+import org.springframework.util.StringUtils
+
+/**
+ * Created by zwali on 13/04/2017.
+ */
+@Repository
+class ItemRepositoryImpl implements ItemRepository {
+    private Table table
+
+    @Value('${amazon.dynamodb.endpoint}')
+    String amazonDynamoDBEndpoint
+
+    @Value('${amazon.aws.accesskey}')
+    String amazonAWSAccessKey
+
+    @Value('${amazon.aws.secretkey}')
+    String amazonAWSSecretKey
+
+    ItemRepositoryImpl(){
+        AmazonDynamoDB amazonDynamoDB = new AmazonDynamoDBClient(new BasicAWSCredentials(amazonAWSAccessKey, amazonAWSSecretKey));
+        if (!StringUtils.isEmpty(amazonDynamoDBEndpoint)) {
```

Comment:
If the string is empty, we should possibly thrown an exception here? Rather than make it look like it is working? What do you think?